### ISSUE TITLE
Some characters in stopwords_tr do not appear Turkish character

### ISSUE THREAD

**Issue by erkanozhan** [quanteda/stopwords#15]
```
stopwords_tr <- data.frame(word = stopwords::stopwords("tr",source="stopwords-iso"), stringsAsFactors = FALSE)
stopwords_tr
```

Some characters in stopwords_tr are not in Turkish. For example;

```
1   acaba
2   acep
3   adamakıllı
4   adeta
5   ait
6   altm**ýþ**   <-Here Must be: **altmış**
7   altmış
8   alt**ý**     <-Here Must be: **altı**

```
I'm looking for a way to fix them.

`stopwords_tr$word<-gsub("ý","ı",stopwords_tr$word)`

The result has not changed. I tried these, but it didn't.

```
Encoding (stopwords_tr $ word) <- "WINDOWS-1254"
Encoding (stopwords_tr $ word) <- "LATIN-5"
Encoding (stopwords_tr $ word) <- "UTF-8"

```
Another interesting thing.

When you double-click stopwords_tr in R Studio to display it, the character appears "**ý**". In Console, it looks like "**y**".
Is there a parameter to set encoding?

Thanks to everyone.

**Comment by kbenoit** [comment]
Thanks for finding this! The “encoding bit” for a character (object) in R can only be one of “Unknown”, “UTF-8”, or Latin-1, but all of the stopwords values should be UTF-8, so we just need to correct the mis-encoded stop words.  I will fix this if you send me the corrections for the words that need it.

Thanks!

**Comment by kbenoit** [comment]
@erkanozhan pls see https://github.com/quanteda/stopwords/pull/16, does that solve it?

**Comment by erkanozhan** [comment]
I'm very glad for your answer. I'm making corrections. I'il send it in a short time. I prepared an excel table. Where can I send the file to you?

**Comment by kbenoit** [comment]
kbenoit@lse.ac.uk but much better to use this tool https://www.tablesgenerator.com/markdown_tables so you can paste the Markdown here. I just really need the wrong words + their corrections, e.g. 

| Wrong  | Corrected |
|--------|-----------|
| altý   | altı      |
| altmýþ | altmış    |

**Comment by erkanozhan** [comment]
I've added fixes to  #16 .
